patches/chrome-android-java-res-layout-new_tab_page_incognito.xml.patch

```diff
@@ -0,0 +1,37 @@
+diff --git a/chrome/android/java/res/layout/new_tab_page_incognito.xml b/chrome/android/java/res/layout/new_tab_page_incognito.xml
+index b294fa17a33e3c089babc00ca4d17b2a62a9e8f3..71ab21408823fc4cf03b3cf1ead406b89f881021 100644
+--- a/chrome/android/java/res/layout/new_tab_page_incognito.xml
++++ b/chrome/android/java/res/layout/new_tab_page_incognito.xml
+@@ -3,7 +3,7 @@
+      Use of this source code is governed by a BSD-style license that can be
+      found in the LICENSE file. -->
+ 
+-<org.chromium.chrome.browser.ntp.IncognitoNewTabPageView
++<org.chromium.chrome.browser.ntp.BraveIncognitoNewTabPageView
+     xmlns:android="http://schemas.android.com/apk/res/android"
+     xmlns:tools="http://schemas.android.com/tools"
+     android:id="@+id/ntp_content"
+@@ -82,8 +82,22 @@
+                 android:textAppearance="@style/TextAppearance.IncognitoNewTabLearnMoreLinkModern"
+                 android:lineSpacingExtra="@dimen/md_incognito_ntp_line_spacing" />
+ 
++            <TextView
++                android:id="@+id/ddg_offer_link"
++                android:layout_width="wrap_content"
++                android:layout_height="wrap_content"
++                android:gravity="center_vertical"
++                android:text="@string/ddg_offer_link"
++                android:textColor="@color/modern_grey_400"
++                android:drawableStart="@drawable/duckduckgo"
++                android:drawablePadding="15dp"
++                android:layout_marginTop="15dp"
++                android:textSize="28sp"
++                android:scaleX="0.5"
++                android:scaleY="0.5" />
++
```

Comment:
Hoping there's a better approach I can take here to avoid such a large patch...

Reply:
@emerick you can use `include` to include a layout inside a different layout https://github.com/brave/brave-core/blob/master/patches/chrome-android-java-res-layout-toolbar_phone.xml.patch#L17

Reply:
Perfect, done.